kr.js

```diff
@@ -1,15 +1,15 @@
 module.exports = `
-menu.btn.ranked 랭크전
+menu.btn.ranked 랭킹전
 menu.btn.shop 상점
-menu.btn.social 허브
+menu.btn.social 소셜
 menu.btn.maps 사용자 지정 게임
-menu.btn.mods Mod
+menu.btn.mods 모드
 menu.btn.settings 설정
-menu.btn.customize 개인 설정
```

Comment:
than maybe we can change to 커스텀 설정

Reply:
We could use 커스터마이징 since Koreans have gotten used to the transliterated word. 설정 has a meaning that is too broad, and 커스텀 설정 sounds like something unrelated to loadout customization.